src/streamlink/stream/ffmpegmux.py

```diff
@@ -130,10 +139,13 @@ def __init__(self, session, *streams, **options):
             self.errorlog = devnull()
 
     def open(self):
-        for t in self.pipe_threads:
-            t.daemon = True
-            t.start()
-        self.process = subprocess.Popen(self._cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=self.errorlog)
+        if self.is_muxed:
+            for t in self.pipe_threads:
+                t.daemon = True
+                t.start()
+
+        self.process = subprocess.Popen(self._cmd, stdout=subprocess.PIPE,
```

Comment:
But I think that `stdin=None` is correct in this case, because the argument to `-i` is a **string** and not a **file object**, like with muxed streams?

Look here [https://docs.python.org/2/library/subprocess.html](https://docs.python.org/2/library/subprocess.html):

> Popen.stdin
> If the stdin argument was PIPE, this attribute is a file object that provides input to the child process. Otherwise, it is None.